Mono.Addins.Setup/Mono.Addins.Setup/SetupTool.cs

```diff
@@ -1057,6 +1075,18 @@ void CreateCommands ()
 			cmd.Description = "Lists available add-in updates.";
 			cmd.AppendDesc ("Prints a list of available add-in updates in the registered repositories.");
 			commands.Add (cmd);
+
+			cmd = new SetupCommand(cat, "enable", "e", new SetupCommandHandler (EnableAddins));
+			cmd.Description = "Enables addins.";
+			cmd.Usage = "<id> ...";
+			cmd.AppendDesc("Enables an add-in repository which has been disabled");
```

Comment:
This should not have repository in the description. It disables/enables addins. Same below